kafka-ui-api/src/main/java/com/provectus/kafka/ui/kafka/KafkaService.java

```diff
@@ -163,9 +182,16 @@ private InternalClusterMetrics collectTopicsMetrics(Map<String,InternalTopic> to
                         c -> {
                             InternalClusterMetrics.InternalClusterMetricsBuilder metricsBuilder = InternalClusterMetrics.builder();
                             metricsBuilder.brokerCount(brokers.size()).activeControllers(c != null ? 1 : 0);
-                            // TODO: fill bytes in/out metrics
+                            Map<String, String> bytesInPerSec;
+                            Map<String, String> bytesOutPerSec;
+                            var jmx = kafkaJmxDto.getJmxParams().stream().filter(j -> j.getBrokerId() == c.id()).findFirst().orElseThrow();
```

Comment:
You should get broker addresses from kafka cluster and use preconfigured jmx port